docs/index.rst

```diff
@@ -151,6 +151,7 @@ Get Started
    :maxdepth: 2
    :hidden:
 
+   upgrade/2.8.0_to_2.9.0.rst
```

Comment:
I'd suggest removing the oldest guide while we're here :)

Reply:
Good idea!